src/components/Login/LoginButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { makeStyles, Button } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  addButton: {
+    padding: '12px 23px',
+  },
+});
+
+const LoginButton = () => {
```

Comment:
is it possible to use our common button for this? seems like it's not doing anything fancy

Reply:
not sure if we have a common button that handles this case - I only see a `BackButton` and a `CancelButton` in our common components folder.